samples/bluetooth/llpm/src/main.c

```diff
@@ -284,11 +289,7 @@ static bool on_vs_evt(struct net_buf_simple *buf)
 	}
 
 	evt = (void *)buf->data;
-	if ((evt->event_counter & 0x3FF) == 0) {
-		printk("QoS conn event reports: "
-		       "channel index 0x%02x, CRC errors 0x%02x\n",
-		       evt->channel_index, evt->crc_error_count);
-	}
+	llpm_latency.crc_errors += evt->crc_error_count;
```

Comment:
For the QoS event, actually seeing the CRC errors per channel is very interesting, because that allows you to change the channel map if there are "bad channels" (higher error count). Should we make the `llpm_latency.crc_errors` an array and print errors per channel instead of total count? On the flip side, that's a lot of channels to print. I'm not sure how to print it in a nice way for this sample.

Reply:
How about using a channel mask as the patch? Set index to channel mask if there is a CRC error. If it is too strict, We can also specify a meaningful value of CRC errors to compare with.

Reply:
Discussed offline. Adding channel bitmask has the downside that once a channel is "bad" it will remain so even though it might only have 1 CRC error. Additionally there might be another channel with 4000 errors. We'll do per channel CRC error reporting as a potential improvement later.